opencti-platform/opencti-graphql/src/schema/schema-attributes.ts

```diff
@@ -105,6 +143,16 @@ export const schemaAttributesDefinition = {
           entityType
         });
       }
+      // Check sortBy on object
+      if (attribute.type === 'object' && attribute.format !== 'flat' && attribute.sortBy) {
```

Comment:
why `attribute.format !== 'flat'` ?

Reply:
because it concerns non flat objects (nested / standard)

Reply:
flat objects cannot define a sortBy

Reply:
we could sort on flatten objects fields, but only as keywords (no support for numeric fields) https://www.elastic.co/guide/en/elasticsearch/reference/current/flattened.html#supported-operations
But it's not the subject here, we could say that we don't support sorting on flatten objects for now.